src/_data/sites/hi.yml

```diff
@@ -0,0 +1,521 @@
+    link: docs
+  - text: दुकान
+    link: store
+    target: _blank
+  - text: Playground
+    link: playground
+    target: _blank
+
+#------------------------------------------------------------------------------
+# Footer
+#------------------------------------------------------------------------------
+
+footer:
+  title: अपना JavaScript कोड ठीक करने के लिए तैयार हैं?
+  description: npm से इंस्टॉल करें या आज ही दान करना शुरू करें।
+  secondary: Secondary
+  social_icons:
+    title: सामाजिक मीडिया
+    twitter: Twitter
+    chat: Discord
+    mailing_list: Google Group
+    github: GitHub
+  theme_switcher:
+    title: Theme Switcher
+    light: Light
+    dark: Dark
+  language_switcher:
+    title: Language Switcher
+    description: एक भाषा का चयन आपको उस भाषा में ESLint वेबसाइट पर ले जाएगा।
+    change_language: भाषा बदलें
+    language: भाषा
+  copyright: >
+    Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and ESLint 
+    contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS 
+    Foundation</a> has registered trademarks and uses trademarks.  For a list of 
+    trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please 
+    see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> 
+    and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  
+    Trademarks and logos not indicated on the 
+    <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation 
+    trademarks</a> are trademarks&trade; or registered&reg; trademarks of 
+    their respective holders. Use of them does not imply any affiliation with 
+    or endorsement by them.
+  links:
+    open_jsf: The OpenJS Foundation
+    terms: उपयोग की शर्तें
+    privacy: गोपनीयता नीति
+    bylaws: OpenJS Foundation Bylaws
+    trademark: Trademark नीति
+    trademark_list: Trademark सूची
+    cookies: Cookie नीति
+
+#------------------------------------------------------------------------------
+# 404 Page
+#------------------------------------------------------------------------------
+
+404_page:
+  title: 404 ग़लती
+  subtitle: पृष्ठ नहीं मिला
+  description: क्षमा करें, आप जिस पृष्ठ की तलाश कर रहे हैं वह मौजूद नहीं है या स्थानांतरित कर दिया गया है।
+  actions:
+    back_to_home: वापस मुख्य पृष्ठ पर
+    browse_docs: दस्तावेज़ ब्राउज़ करें
+
+#------------------------------------------------------------------------------
+# Branding Page
+#------------------------------------------------------------------------------
+
+branding_page:
+  title: Branding
+  description: हमारे लोगो का उपयोग कैसे करें
+  sections:
+    name:
+      title: नाम
+      description: >
+        ESLint को उपयोग किए जाने पर बड़े अक्षर E, S और L के साथ लिखा जाना चाहिए, क्योंकि "ES" का अर्थ ECMAScript और "L" शब्द "Lint" की शुरुआत है।
+    logo:
+      title: Logo
+      description:
+        - >
+          ESLint लोगो को विभिन्न backgound में रखा जा सकता है, बशर्ते इसमें पर्याप्त लंबवत और क्षैतिज पैडिंग हो।
```

Comment:
@snitin315 - I guess it should be `background` but not `backgound`.

Reply:
Yeah! thanks for spotting it.